cs/cli/vowpalwabbit.h

```diff
@@ -40,6 +40,10 @@ public ref class VowpalWabbit : VowpalWabbitBase, IVowpalWabbitExamplePool
   template<typename T>
   cli::array<cli::array<float>^>^ FillTopicAllocation(T& weights);
 
+  // Cached empty example.  
+  // This is used to emit empty lines to cache while handling multiline examples
+  void CacheEmptyLine();
```

Comment:
should this be private? Not I'm even clear on when to call this and it seems Learn/Predict(multi line) takes care of it.

Also please use c++/cli comment style

Reply:
This is used to emit empty lines to cache while handling multiline examples.  Used internally by Learn(IEnumerable<String> lines)

Reply:
Changed comment style